sample-code/src/aiservice.ts

```diff
@@ -1,22 +1,14 @@
-import { OpenAiClient } from '@sap-ai-sdk/gen-ai-hub';
+import { OpenAiClient, OpenAiModels } from '@sap-ai-sdk/gen-ai-hub';
 
 const openAiClient = new OpenAiClient();
 
-const deployments: { [model: string]: string } = {
-  'gpt-4-32k': 'd577d927380c98ea',
-  'gpt-35-turbo': 'd66d1927bf590375',
-  ada: 'd0084a63ebd7bcd3'
-};
-
 /**
  * Ask GPT about the capital of France.
  * @returns The answer from GPT.
  */
```

Comment:
As a certified `.then` hater, and for consistency, I would transform all these function to async / await functions
```ts
export async function chatCompletion(): Promise<any> {
return (await openAiClient
    .chatCompletion(OpenAiModels.GPT_35_TURBO, {
      messages: [{ role: 'user', content: 'What is the capital of France?' }]
    })).choices[0].message.content
}
```

Reply:
Also the any type seems a bit wonky

Reply:
Fixed ✅